### PR TITLE
README: further cleanups after rename of utils -> sdk

### PR DESCRIPTION
**Pull request by csett86**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,16 +64,6 @@ Install Node.js 16 first (or if you use [nvm](https://github.com/nvm-sh/nvm), sw
 
 ```bash
 npm install --global --production windows-build-tools
-npm config set msvs_version 2017
-```
-</details>
-
-<details><summary>Extra dependencies for GNU/Linux</summary>
```
Comment (saghul) [inline README.md:71]: How about we keep all of this, but clarify it's only needed for development?

**Review by saghul — APPROVED** [review]
LGTM with a suggestion.

File: README.md (hunk 1 above)
Comment (csett86) [reply README.md]: Agreed, have added them back in

Comment (csett86) [reply README.md]: Only the `npm config set msvs_version 2017` is no longer necessary, thus removing that while at it.

**Review by saghul — APPROVED** [review]

**Pull request merged**